webui/react/src/utils/types.ts

```diff
@@ -36,7 +36,7 @@ export const experimentToTask = (experiment: ExperimentItem): RecentExperimentTa
     resourcePool: experiment.resourcePool,
     startTime: experiment.startTime,
     state: experiment.state,
-    url: experiment.url,
+    url: `/experiments/${experiment.id}`,
```

Comment:
I should've just tried it, but I was worried that it might create a weird dependency loop. Another thing is hoping to get rid of the `url` property whenever possible since we have the route/utils `paths` (not for this one quite yet).